host/src/components/GameInProgressContentSwitch.jsx

```diff
@@ -235,12 +235,12 @@ export default function GameInProgressContentSwitch ({
 
   const questionCofigurationComponents = [
     <Box className={classes.configContainer}>
-      <div id="questioncard-scrollbox" ref={questionCardRef}>
+      {/* <div id="questioncard-scrollbox" ref={questionCardRef}>
```

Comment:
When we pull out the question card, we'll also need to adjust the nav bar in the footer:
<img width="707" alt="Screenshot 2023-12-26 at 10 51 13" src="https://github.com/rightoneducation/righton-app/assets/79417944/76376769-9db5-4c96-bb72-1d0856eb9732">

(note that the dropdown says question card still and it will throw an exception if we try to select it). 

I think we can just delete this instead of commenting it out and then adjust the nav bar accordingly so there isn't a question card option.